geonode_mapstore_client/client/js/epics/gnresource.js

```diff
@@ -135,7 +136,7 @@ const resourceTypes = {
                 ])
                     .then((response) => {
                         const [, gnLayer] = response ?? [];
-                        if (gnLayer?.has_time) {
+                        if (gnLayer?.has_time && !isCataloguePage) {
```

Comment:
Why we need to check the catalogue page?
In which other case this is workflow is triggered?

Reply:
see https://github.com/GeoNode/geonode-mapstore-client/pull/2017#discussion_r2057759451